tests/parametric/test_dynamic_configuration.py

```diff
@@ -690,6 +690,7 @@ def test_trace_sampling_rules_override_rate(self, library_env, test_agent, test_
         reason="JSON tag format in RC differs from the JSON tag format used in DD_TRACE_SAMPLING_RULES",
     )
     @bug(context.library == "ruby", reason="RC_SAMPLING_TAGS_RULE_RATE is not respected")
+    @bug(context.library == "dotnet", reason="Not applying correct sampling rate")
```

Comment:
Can we remove this line since you've updated the dotnet manifest file?

Reply:
Not yet 😞  In the manifest I enabled all the tests in class `TestDynamicConfigSamplingRules`, but this line still skips one of them because it is failing, `TestDynamicConfigSamplingRules::test_trace_sampling_rules_with_tags`.

I wanted to enable the rest of the tests while I investigate this particular test failure. It fails to match remote sampling rules with tags, which is still not supported by the frontend/backend, so not an urgent blocker.

Reply:
Can we do this in the manifest file?

Reply:
Hmm. In that case I recommend doing something like:
```suggestion
    @bug(context.library <= "dotnet@2.53.2", reason="Not applying correct sampling rate")
```

Reply:
`context.library <= "dotnet@2.53.2"` means we fixed this test in `2.53.2`, but we haven't fixed it yet. The _other_ tests in that class start passing with `2.53.2`. This one test still fails and requires investigation.

Reply:
Effectively, what I want to do is something like this in the manifest file (but I don't think this is valid?)
```yaml
tests/:
  parametric/:
    test_dynamic_configuration.py:
      TestDynamicConfigSamplingRules:
        test_trace_sampling_rules_with_tags: bug
        test_trace_sampling_rules_override_env: v2.53.2
        test_trace_sampling_rules_override_rate: v2.53.2
        test_remote_sampling_rules_retention: v2.53.2
```

Reply:
Two things are needed to enable this one test:
- Sampling rules were not matching by tags because we were not adding tags at all in .NET. In this PR I add support in the .NET driver app used for parametric tests.
- All local sampling rules should be ignored is there are any remote rules present (even if none of them match the tracer). I have a fix for this here:
https://github.com/DataDog/dd-trace-dotnet/pull/5720
I will enable this test when that PR is merged.